src/app/home/home-page.component.ts

```diff
@@ -7,5 +8,25 @@ import { Component } from "@angular/core";
 })
 
 export class HomePageComponent {
-  title = "Travel Track"
+  // FINISH YOUR BUCKET LIST: box 1 details
```

Comment:
I think these box titles are stub data?

Maybe you'll be pulling this information from an API in the future? If so, you should make it look like you're doing that now by using a service that returns an `Observable<SomeType>`.

Maybe this is static text on the page. If so, your static text should try to stay in HTML as much as possible. This helps with translation tools and reduces clutter in the component.